anecdote/assets/logs/Anecdote Log Pipeline.yaml

```diff
@@ -0,0 +1,10 @@
+id: anecdote
+metric_id: anecdote
+facets: null
+pipeline:
+  type: pipeline
+  name: Anecdote Log Pipeline
+  enabled: true
+  filter:
+    query: "source:anecdote "
+  processors: []
```

Comment:
There are not processors or facets in the pipeline. In order to approve this integration, you must have a log pipeline. Please follow our documentation on [how to create a log pipeline. ](https://docs.datadoghq.com/developers/integrations/log_pipeline/)

Reply:
I updated the log pipeline

Reply:
The log pipeline requires all fields that match standard attributes to be remapped. Additionally, it is recommended to include facets. Please update the pipeline based off our docs and your sample.

Reply:
Please check it again, all tests are succeeded